### PR TITLE
RAC-1215: fix error when attributeOption doesn't have label

### PR DESCRIPTION
**Pull request by MadeWilson**

### CODE REVIEW

File: src/Akeneo/Pim/Structure/Bundle/Query/PublicApi/AttributeOption/Sql/SqlSearchAttributeOptions.php [hunk 1]
```diff
@@ -95,7 +95,7 @@ private function getAttributeOptions(
         return array_map(
             static fn (array $attributeOption) => new AttributeOption(
                 $attributeOption['code'],
-                json_decode($attributeOption['labels'], true),
+                null !== $attributeOption['labels'] ? json_decode($attributeOption['labels'], true) : [],
```
Comment (pierrejolly) [inline src/Akeneo/Pim/Structure/Bundle/Query/PublicApi/AttributeOption/Sql/SqlSearchAttributeOptions.php:98]: Shouldn't we return the code between brackets when the label is empty?

Comment (MadeWilson) [reply src/Akeneo/Pim/Structure/Bundle/Query/PublicApi/AttributeOption/Sql/SqlSearchAttributeOptions.php:98]: no the code is already given no matter what. In our component we have a method getLabel() that return the code if no label exist

**Review by ghoussard — APPROVED** [review]
Could you add a test case in tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php to cover this case please ?

**Comment by MadeWilson** [comment]
> Could you add a test case in tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php to cover this case please ?

sure

**Review by StevenVAIDIE — APPROVED** [review]

**Pull request merged**